articles/active-directory/develop/quickstart-v2-android.md

```diff
@@ -19,7 +19,8 @@ ms.custom: aaddev, identityplatformtop40, scenarios:getting-started, languages:A
 
 This quickstart uses a code sample to demonstrate how an Android application can sign in personal, work, or school accounts using the Microsoft identity platform, and then get an access token and call the Microsoft Graph API.
 
-Applications must be represented by an app object in Azure Active Directory so that the Microsoft identity platform can share tokens with your application.
+Applications must be represented by an app object in Azure Active Directory so that the Microsoft identity platform can provide
+tokens to your application.
```

Comment:
```suggestion
Applications must be represented by an app object in Azure Active Directory so that the Microsoft identity platform can provide tokens to your application.
```